packages/main/src/TimeSelectionClocks.hbs

```diff
@@ -16,13 +17,14 @@
 		{{/if}}
 		<ui5-toggle-spin-button
 			id="{{../_id}}_button_{{this.entity}}"
+			data-sap-clock="{{this.entity}}"
 			.valueMin="{{this.attributes.min}}"
 			.valueMax="{{this.attributes.max}}"
 			.valueNow="{{this.value}}"
 			.valueText="{{this.textValue}}"
 			.accessibleName="{{this.label}}"
 			.pressed="{{this.active}}"
-			?focused="{{this.active}}"
+			?focused="{{this.focused}}"
```

Comment:
Do we have focused property? I see that we have _focused, but not focused.

Reply:
We do, but as property of TimePickerEntityProperties type (PimePickerInternals.ts), and this part renders such entity here. Its 'focused' is calculated a combination of  mentioned '_focused' property and '_amPmFocused' property.